docs/resources.md

```diff
@@ -359,9 +358,49 @@ service account.
          secretName: bucket-sa
          secretKey: service_account.json
    ```
-
 ---
 
+#### BuildGCS Storage Resource
+BuildGCS storage resource points to
+[Google Cloud Storage](https://cloud.google.com/storage/) blob like  [GCS Storage Resource](#gcs-storage-resource) either in the form of a .zip archive, or based on the contents of a source manifest file.
+In addition to fetching an .zip archive, BuildGCS also unzips it.
+A [Source Manifest File](https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/gcs-fetcher#source-manifests) is a JSON object listing other objects in Cloud Storage that should be fetched. The format of the manifest is a mapping of destination file path to the location in Cloud Storage where the file's contents can be found.
```

Comment:
usually we wrap long lines in markdown files (see [the bot formatting](https://github.com/knative/build-pipeline/pull/504))